godtossing/Pos/Ssc/GodTossing/Toss/Base.hs

```diff
@@ -464,6 +468,11 @@ checkSharesPayload epoch shares = do
 -- For certificates we check that
 --   * certificate hasn't been sent already
 --   * certificate is generated by richman
+--   * there is no existing certificate with the same VSS key in
+--     MonadToss's state
+--   * all certificates in the 'VssCertificatesMap' have distinct VSS keys
+--      (this check is useless for certificate maps obtained through
+--      deserialization, but not useless in other cases)
```

Comment:
Do we still need this check here, given that it's now invariant of `VssCertificatesMap` type?

Reply:
Shouldn't be needed.